source/speech/sayAll.py

```diff
@@ -104,8 +111,19 @@ def readText(
 		startPos: Optional[textInfos.TextInfo] = None,
 		nextLineFunc: Optional[Callable[[textInfos.TextInfo], textInfos.TextInfo]] = None,
 		shouldUpdateCaret: bool = True,
+		startedFromScript: bool | None = False,
 	) -> None:
+		"""Start or restarts the reader
+		:cursor: the type of cursor used for say all
+		:startPos: start position (only used for table say all)
+		:nextLineFunc: function called to read the next line (only used for table say all)
```

Comment:
@mltony some of the parameters of this functions have been introduced with table say all. Does the description seems OK to you or would you suggest something different or more precise? Thanks.